src/xcode/fastlane/Snapfile

```diff
@@ -1,56 +0,0 @@
-# Uncomment the lines below you want to change by removing the # in the beginning
```

Comment:
Is this file not needed anymore?

Reply:
Nope its all inside the normal Fastfile. We had a split here before, some options set in the Snapfile and some in the Fastfile. I cleaned this up by moving it all to the Fastfile.